jobs/ci-kubernetes-e2e-kops-aws-release-1.5.env

```diff
@@ -0,0 +1 @@
+KOPS_DEPLOY_LATEST_URL=https://storage.googleapis.com/kubernetes-release-dev/ci/latest-1.5.txt
```

Comment:
My theory was that I would layer these, by passing multiple env args.  So I would do:
```
    "--env-file=platforms/kops_aws.env",
    "--env-file=jobs/ci-kubernetes-e2e-kops-aws.env",
    "--env-file=jobs/ci-kubernetes-e2e-kops-aws-release-1.5.env"
```

I'm not sure if that works or is a good idea....